presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/db/FunctionNamespaceDao.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.functionNamespace.db;
+
+import com.facebook.presto.spi.function.RoutineCharacteristics.Language;
+import com.facebook.presto.spi.function.SqlFunctionId;
+import com.facebook.presto.spi.function.SqlInvokedFunction;
+import com.facebook.presto.spi.function.SqlParameter;
+import com.facebook.presto.spi.type.TypeSignature;
+import org.jdbi.v3.sqlobject.config.RegisterArgumentFactories;
+import org.jdbi.v3.sqlobject.config.RegisterArgumentFactory;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMappers;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.Define;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.util.List;
+import java.util.Optional;
+
+@RegisterRowMappers({
+        @RegisterRowMapper(SqlInvokedFunctionRowMapper.class),
+        @RegisterRowMapper(SqlInvokedFunctionRecordRowMapper.class)
+})
+@RegisterArgumentFactories({
+        @RegisterArgumentFactory(SqlFunctionIdArgumentFactory.class),
+        @RegisterArgumentFactory(SqlParametersArgumentFactory.class),
+        @RegisterArgumentFactory(TypeSignatureArgumentFactory.class),
+})
+public interface FunctionNamespaceDao
+{
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS <table_name> (\n" +
```

Comment:
Maybe name it `<function_namespaces_table_name>` to make the query clearer to read.